configure/configuring-fairsharing.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="configuring-fairsharing"]
+= Configuring fair sharing
+:context: configuring-fairsharing
+
+toc::[]
+
+Fair sharing is a preemption strategy that is used to achieve an equal or weighted share of borrowable resources between the tenants of a cohort. Borrowable resources are the unused nominal quota of all the cluster queues in a cohort.
+
+You can configure fair sharing by xref:../install/install-kueue.adoc#create-kueue-cr_install-kueue[setting the `spec.config.preemptionPolicy` value in the `Kueue` custom resource (CR) to `FairSharing`].
```

Comment:
sorry, this is not quite correct either..

spec.config.preemption.preemptionPolicy

Reply:
lol I was wondering actually 😂 np I'll update it